### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Delete_DeletePopup_Screenreader : screen reader does not announce any successful message that the blob is deleted.

### ISSUE THREAD

**Issue by prasadaps** [microsoft/AzureStorageExplorer#3469]
## Bug Description ##
GitHubTags: #C&AI_Aug2020_BMAzure Tools Storage Explorer;#A11yWCAG2.1;#A11YTCS;#DesktopApp;#Azure Tools Storage Explorer;#WCAG4.1.3;#JAWS;#NVDA;#Win32;#A11ySev2;#Benchmark;

### Environment Details:
Application Name: Azure Tool Storage Explorer
Application version:  1.14.2
Windows Version - Windows10 
Screen reader: JAWS(2020.1912.11ILM), NVDA Version:2020.2

### Repro Steps:
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Blob Containers" of the selected subscription, right click and hit enter on "Create Blob container" to Create a new Blob container.
5. Once completed, Navigate to "Upload" button and Upload a blob/file by using  "Upload Files" control.
6. Once Completed, Right click on uploaded blob and Select "Delete" and hit enter.
7. Start screen reader and press OK. and observe there is no message announce for screen reader users for deletion of blob is completed.

### Actual:
When users delete any blob the screen reader does not announce any successful message that the blob is deleted for screen reader users .

Note : This issue is observed with both JAWS and NVDA screen reader.

### Expected:

As soon as the blob is deleted the screen reader should announce status message of blob is deleted.

### User Impact: 
The screen reader users will not get any acknowledge that the blob deletion in successful.

### Attachment
https://microsoft-my.sharepoint.com/:v:/p/v-rojad/EcZuFciPohRJoMXXRnyDImYBVhtdnD9BxDOfDW3m9K5VIg?e=NYpyc9

**Comment by MRayermannMSFT** [comment]
@prasadaps please retest against Storage Explorer version 1.15.1 or greater. Can be re downloaded from https://storageexplorer.com

I can not repro that issue in version 1.15.1.
![image](https://user-images.githubusercontent.com/21112697/92957961-45796000-f41e-11ea-9efd-a1545e1b15f9.png)

![image](https://user-images.githubusercontent.com/21112697/92958030-60e46b00-f41e-11ea-9207-2df38cb7e44b.png)

**Comment by prasadaps** [comment]
I have verified the bug in latest version of the application this issue is not reproducing there. this issue is fixed.
hence closing the bug.
please refer the attachment - https://microsoftapc-my.sharepoint.com/:i:/g/personal/v-prphat_microsoft_com/EUl5TIuQAHhOkbJvobmY_ocBOVOB7rPjVc8z0pUf0bQB-Q?e=RF0Zsr

**Comment by 12-shweta** [comment]
Github Tags:#Closed;
